packages/renderer/src/lib/chat/components/chat.svelte

```diff
@@ -51,6 +51,8 @@ let selectedMCP = $state<MCPRemoteServerInfo[]>(
 
 const chatHistory = ChatHistory.fromContext();
 
+let totalTokens = $state(messages[messages.length - 1]?.tokens ? messages[messages.length - 1].tokens : 0);
```

Comment:
$derived

Reply:
Cannot use $derived as the messages array is not updated but client.messages